torch/_dynamo/debug_utils.py

```diff
@@ -656,7 +656,10 @@ def tensor(self, name, t) -> None:
             args.append(str(tuple(t.stride())))
         if _dtype_or_default(None) != t.dtype:
             args.append(f"dtype={t.dtype!r}")
-        if _storage_offset_or_default(None) != t.storage_offset():
+        if (
+            isinstance(t.storage_offset(), torch.SymInt)
+            or _storage_offset_or_default(None) != t.storage_offset()
```

Comment:
statically_known_true is probably better here.

`if not statically_known_true(_storage_offset_or_default(None) == t.storage_offset()): ...`